### DISCUSSION TITLE
Decoding non-Evohome messages?

### DISCUSSION THREAD

**Discussion by samus-homebridge** [ghoti57/evofw3#29] [General]
Hello, I am very much an amateur here and so forgive me if any of this is obviously impossible within this framework!

I currently have an SSM-D running on a RPi and it's only real purpose is to receive temperature readings from an orphaned Honeywell DHW sensor, which then gets sent out to me on a private twitter feed for keeping track while I'm away from home.

I live in Scotland and as of this year we are required to have wireless interlinked smoke/heat alarms throughout the house. I bought Hispec brand alarms which also operate on 868 MHz. Beyond that I have no idea yet about their protocol/packet structure.

What I was hoping to do is reuse the SSM-D to listen for smoke alarms, decoded using something like RTL_433 and then similarly pushed to me by text or twitter or something.  

Is it possible to access the "raw" 868 MHz datastream received my the SSM-D and shunt it off for that separate processing, or is that not possible if the serial output of evofw3 is already decoded into the Evohome packet structure?  Any guidance or suggestions are most welcome!  Thank you.

**Comment by ghoti57** [comment]
The serial output from the SSM-D is a decoded form of the packets used by the Honeywell hardware.

It might be worth looking to see if CULFW can decode the HISPEC messages.

It's also unlikely that the HISPEC protocol will look like a 38400 baud serial connection used by Honeywell.

⁣Get BlueMail for Android ​

**Comment by samus-homebridge** [comment]
Okay, that is not entirely unexpected but thanks for thinking about it!
